apps/system/test_softlockup/test_softlockup.c

```diff
@@ -0,0 +1,189 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <unistd.h>
+#include <tinyara/config.h>
+#include <pthread.h>
+#include <tinyara/wqueue.h>
+
+#ifdef CONFIG_SYSTEM_SOFTLOCKUP
```

Comment:
This file will be built when this config is enabled. So this is redundant.

Reply:
yes. Removed it